melos.yaml

```diff
@@ -71,14 +71,14 @@ scripts:
 
   analyze: >
     melos exec -c 1 --no-private --ignore="*example*" -- \
-      pub global run tuneup check
+      flutter pub global run tuneup check
```

Comment:
optional: If removing the global tuneup installation as suggested in below comment, it might make sense to change this task to use deeper YAML syntax with a description block like https://github.com/aws-amplify/amplify-flutter/blob/master/melos.yaml#L51 and include a note in the description that this requires tuneup installed, and to install it, run `flutter pub global activate tuneup`.